src/cascadia/UnitTests_TerminalCore/SelectionTest.cpp

```diff
@@ -628,60 +628,60 @@ namespace TerminalCoreUnitTests
-                ValidateLinearSelection(term, { 4, 10 }, { 99, 10 });
+                ValidateLinearSelection(term, { 4, 10 }, { term.GetViewport().RightExclusive(), 10 });
             }
 
-            // Step 5: Shift+Double-Click at "dragThroughHere"
+            Log::Comment(L"Step 5: Shift+Double-Click at \"dragThroughHere\"");
             {
                 // Simulate Shift+DoubleClick at (x,y) = (21,10)
                 //
-                // buffer: doubleClickMe dragThroughHere
-                //         ^                ^          ^
-                //       start            click      finish
+                // buffer: doubleClickMe dragThroughHere anotherWord
+                //         ^                ^            ^
+                //       start            click        finish
                 term.SetSelectionEnd({ 21, 10 }, Terminal::SelectionExpansion::Word);
 
                 // Validate selection area: "doubleClickMe dragThroughHere" selected
-                ValidateLinearSelection(term, { 4, 10 }, { 32, 10 });
+                ValidateLinearSelection(term, { 4, 10 }, { 33, 10 });
             }
 
-            // Step 6: Drag past "dragThroughHere"
+            Log::Comment(L"Step 6: Drag past \"dragThroughHere\"");
             {
                 // Simulate drag to (x,y) = (35,10)
                 // Since we were preceded by a double-click, we're in "word" expansion mode
                 //
-                // buffer: doubleClickMe dragThroughHere     |
+                // buffer: doubleClickMe dragThroughHere anotherWord
                 //         ^                                 ^
                 //       start                             finish (boundary)
                 term.SetSelectionEnd({ 35, 10 });
 
                 // Validate selection area: "doubleClickMe dragThroughHere..." selected
-                ValidateLinearSelection(term, { 4, 10 }, { 99, 10 });
+                ValidateLinearSelection(term, { 4, 10 }, { gsl::narrow<til::CoordType>(4 + text.size()), 10 });
             }
 
-            // Step 6: Drag back to "dragThroughHere"
+            Log::Comment(L"Step 7: Drag back to \"dragThroughHere\"");
             {
                 // Simulate drag to (x,y) = (21,10)
                 //
-                // buffer: doubleClickMe dragThroughHere
-                //         ^                ^          ^
-                //       start             drag      finish
+                // buffer: doubleClickMe dragThroughHere anotherWord
+                //         ^                ^           ^
+                //       start             drag       finish
                 term.SetSelectionEnd({ 21, 10 });
 
-                // Validate selection area: "doubleClickMe dragThroughHere" selected
-                ValidateLinearSelection(term, { 4, 10 }, { 32, 10 });
+                // Validate selection area: "doubleClickMe dragThroughHere " selected

```

Comment:
q about space again

Reply:
this one seems like *yes* land on the space, since that's where `finish` is

Reply:
33 points to the space, but since we're exclusive, we're not actually selecting the space